client/package.json

```diff
@@ -158,10 +158,10 @@
   },
   "scripts": {
     "vscode:prepublish": "npm run compile",
-    "compile": "tsc -p ./",
-    "watch": "tsc -watch -p ./",
+    "compile": "rimraf out && tsc -p ./",
+    "watch": "rimraf out && tsc -watch -p ./",
```

Comment:
Required to clean out any compiled files that have now had their corresponding source file deleted.